src/scheduler/analysis/plugin.py

```diff
@@ -206,10 +206,12 @@ def _handle_sigterm(signum, frame):
             except queue.Empty:
                 continue
 
+            analysis_id = f'{self._plugin.metadata.name} analysis on {task.scheduler_state.uid}'
```

Comment:
nit: This is not really an id. More like analysis description or so.

Reply:
changed